.github/workflows/clair-in-ci.db.yaml

```diff
@@ -29,38 +30,29 @@ jobs:
           fi
 
       - name: Log into registry ${{ env.REGISTRY }}
-        uses: docker/login-action@v2.2.0
+        uses: redhat-actions/podman-login@v1
         if: ${{ github.event.client_payload.pr_event != 'pull_request' }}  # don't login from PR; secrets are not passed to PRs from fork
         with:
           registry: ${{ env.REGISTRY }}
           username: ${{ secrets.HACBS_TEST_QUAY_USER }}
           password: ${{ secrets.HACBS_TEST_QUAY_TOKEN }}
 
-      - name: Setup docker for workflow
+      - name: Setup podman for workflow
```

Comment:
I'm not sure what's the purpose of this for docker, and I'm really not sure if this is needed for podman, mainly we don't build here anything anymore

Reply:
I'm not sure either, I kept it in to be safe.  I can get rid of it.